src/creature.h

```diff
@@ -55,6 +55,8 @@ template<typename V, typename U>
 class quantity;
 class mass_in_gram_tag;
 using mass = quantity<int, mass_in_gram_tag>;
+class volume_in_milliliter_tag;
+using volume = quantity<int, volume_in_milliliter_tag>;
```

Comment:
And why did you add those two lines?

Reply:
Notice line 56-57 where similar lines for mass were already added?
It prevents this compile error:
![image](https://user-images.githubusercontent.com/39925111/47615021-82960c80-daa9-11e8-820b-724633aaad8f.png)